src/renderer/components/header/HeaderComponent.tsx

```diff
@@ -300,6 +303,15 @@ export const HeaderComponent: React.FC<Props> = (props): JSX.Element => {
                   <AsgardexLogo />
                   {renderHeaderNetStatus}
                   <HeaderTheme isDesktopView={isDesktopView} />
+                  {isLargeDesktopView && (
```

Comment:
Show price + volume for large desktops only. In future we will clean up the header to move some settings into `Settings` (e.g. language, theme), but not for this or next week :)